WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsInputDialogFragment.java

```diff
@@ -35,6 +34,8 @@ public interface PostSettingsInputDialogListener {
     private static final String HINT_TAG = "hint";
     private static final String DISABLE_EMPTY_INPUT_TAG = "disable_empty_input";
     private static final String MULTILINE_INPUT_TAG = "is_multiline_input";
+
+    private PostSettingsInputDialogBinding mBinding;
```

Comment:
Suggestion (💡): When possible, it is best to not utilize such a view `binding` field. You could instead keep the whole usage locally to the `onCreateDialog(...)` function and his way to avoid all the boilerplate, including removing the `binding` reference during `onDestroy()`.

Reply:
Makes sense!! made the changes.

Reply:
Great, thanks for this change @neeldoshii !

Minor (🔍): Can you also please rename `mBinding` to `binding`. This `mBinding` is no longer a field, but rather a local variable, and as such we should respect the guidance with those.